docs/releases/5.0.txt

```diff
@@ -576,6 +576,10 @@ Miscellaneous
 * The minimum supported version of SQLite is increased from 3.21.0 to 3.27.0.
 
 * Support for ``cx_Oracle`` < 8.3 is removed.
+* ``django.utils.text.Truncator`` used by :tfilter:`truncatechars_html` and 
+  :tfilter:`truncatewords_html` now uses an HTML parser which is more 
+  robust. HTML entities are now decoded and there may be other small 
```

Comment:
I'm not sure what this sentence means, what would be a decoded HTML entity? :thinking:

Reply:
I think I'm just trying to say there may be small differences in the output compared to the previous version. 

The release note is now:

```
* ``django.utils.text.Truncator`` used by :tfilter:`truncatechars_html` and
  :tfilter:`truncatewords_html` now uses a :py:class:`html.parser.HTMLParser` 
  in its underlying implementation. This change results in a more robust and 
  faster operation, but there may be small differences in the output. 

```